pkg/cmd/pr/shared/params.go

```diff
@@ -149,14 +149,25 @@ type FilterOptions struct {
 	BaseBranch string
 	Mention    string
 	Milestone  string
+	Search     string
 }
 
 func ListURLWithQuery(listURL string, options FilterOptions) (string, error) {
 	u, err := url.Parse(listURL)
 	if err != nil {
 		return "", err
 	}
+
+	params := u.Query()
+	params.Set("q", IssueSearchBuild(options))
+	u.RawQuery = params.Encode()
+
+	return u.String(), nil
+}
+
+func IssueSearchBuild(options FilterOptions) string {
```

Comment:
This doesn't seem specific to issues, could we name this `SearchQueryBuild` instead? I expect this function will be used when expanding the search functionality to pull requests and then having issue in the title might be confusing.

Reply:
Certainly @samcoe Updated it.